src/inspector/models/popups/abstractelementpopupmodel.h

```diff
@@ -47,6 +47,7 @@ class AbstractElementPopupModel : public QObject, public async::Asyncable, publi
         TYPE_UNDEFINED = -1,
         TYPE_HARP_DIAGRAM,
         TYPE_CAPO,
+        TYPE_TEXT
```

Comment:
Wouöldn't `STAFF_TEXT `be need here too, IOW: shouldn't this be in sync with `ELEMENT_POPUP_TYPES`

Reply:
No, all types of text elements will get the same popup with the same model, like in the Inspector. So `ELEMENT_POPUP_TYPES` maps all these element types to the `TYPE_TEXT` model type.